src/MSRIOGroup.cpp

```diff
@@ -1176,6 +1288,15 @@ namespace geopm
                     description = field_data["description"].string_value();
                 }
 
+                if (msr_field_name == "QM_EVT_SEL:RMID" || msr_field_name == "PQR_ASSOC:RMID") {
+                    if((end_bit - begin_bit) != m_rdt.rmid_bit_width) {
+                        throw Exception("MSRIOGroup::" + std::string(__func__) + "(): RMID bit width " +
```

Comment:
I think this is fine, but I'd prefer using ostringstream rather than string conversion with adds to generate the error message.